internal/versions/mock_versions.go

```diff
@@ -9,7 +9,7 @@ import (
 	middleware "github.com/go-openapi/runtime/middleware"
 	gomock "github.com/golang/mock/gomock"
 	models "github.com/openshift/assisted-service/models"
-	versions "github.com/openshift/assisted-service/restapi/operations/versions"
+	versions0 "github.com/openshift/assisted-service/restapi/operations/versions"
```

Comment:
Can we be more descriptive with the variable renaming here? This looks like a typo. It would make more sense to add an adjective instead of an index.

Reply:
it's auto generated